openedx/features/course_experience/api/v1/urls.py

```diff
@@ -17,3 +18,12 @@
         name='course-experience-reset-course-deadlines'
     ),
 ]
+
+# URL for retrieving course deadlines info
+urlpatterns += [
+    re_path(
+        r'v1/course_deadlines_info/{}'.format(settings.COURSE_KEY_PATTERN),
+        CourseDeadlinesMobileView.as_view(),
+        name='course-experience-course-deadlines-mobile'
```

Comment:
I assume this follows the URL pattern of other mobile only API?  I'm a little surprised we don't have the word `mobile` in there or something, but whatever.

Reply:
This was the URL provided in the Jira ticket. I'm assuming it's not set in stone, if we think we should include `mobile`?

Reply:
yea that's what I thought would be best in this case, to keep consistent with the `reset_course_deadlines` URL and the mobile team was fine with that.  There are only a subset of the APIs the native app accesses that include the `mobile` token, and also I'm not confident that this endpoint won't be useful at some other point outside of the mobile case